client/components/domains/mixins/domain-suggestion-mixin.jsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * Internal Dependencies
+ **/
+import cartItems from 'lib/cart-values/cart-items';
+
+export default {
+	willBundle( withPlansOnly, selectedSite, cart, suggestion ) {
```

Comment:
We will get rid of `abtest` module, but we are not getting rid of this value since we still need to serve two types of users. The value will be stored in redux tree and will be tied to a user, but not going away.